docs/installation-and-operation/serialization.md

```diff
@@ -12,116 +12,374 @@ Once you really get rolling with Metabase, it's often the case that you'll have
 
 To help you out in situations like this, Metabase has a serialization feature which lets you create an _export_ of the contents of a Metabase that can then be _imported_ into one or more Metabases.
 
+**Export** will serialize your the contents of your source Metabase as YAML files.
```

Comment:
I thing the "will serialize your the contents" needs to change to "will serialize the contents"